evansmusomi/react-native-102
# Stores Picked Location in Redux

*What this does*
- Stores picked location in redux
- Displays picked location on a map in the Place Detail screen

*How to verify*
- Share a place on the Share Place screen
- Notice the Share Place button is disabled until you select a location on the map or locate yourself
- Navigate to the Find Place screen
- Select a shared place to navigate to the Place Detail screen
- You should see a map with a marker pointing to the place's location
